config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   namespace :api, defaults: { format: 'json' } do
     namespace :v1 do
+      delete '/users/:id', to: 'users#destroy'
```

Comment:
I recommend to use the `resources` method to simplify this.
```suggestion
      resources :users, only: :destroy
```